### PR TITLE
02: Creates the buttons component and stories.

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
- [ ] Checkout [this branch of the western-law site](https://github.com/emulsify-ds/western-law/pull/1)
- [ ] Clone this repo into the root of the western-law site, checkout this branch, and run `yarn` to set it up with workspaces.
- [ ] Run `yarn storybook` and verify the buttons are showing up in the storybook ui.

### CODE REVIEW

File: 01-atoms/buttons/Button.component.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component that renders a button with a click handler.
+ */
+const Button = (props) => {
+  const { onClick, children, modifiers } = props;
```
Comment (nJim) [inline 01-atoms/buttons/Button.component.js:13]: I do this both ways, but when I have a short list of props, I will often restructure them within the arguments. Just giving you an option if you like the syntax.
```
const Button = ({ onClick, children, modifiers }) => {
```

File: 01-atoms/buttons/Button.component.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      className={classNames}
+      aria-label="button"
+      onClick={onClick}
+    >
+      {children}
+    </button>
+  );
+};
+
+Button.propTypes = {
```
Comment (nJim) [inline 01-atoms/buttons/Button.component.js:32]: Missing the modifiers array prop type.

File: 01-atoms/buttons/buttons.stories.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ */
+export default { title: 'Atoms/Button' };
+
+export const All = () => (
+  <>
+    <Button>Button Default</Button>
+    <br />
+    <Button modifiers={['alt']}>Button Alt</Button>
+    <br />
+    <Button modifiers={['alt-2']}>Button Alt 2</Button>
```
Comment (nJim) [inline 01-atoms/buttons/buttons.stories.js:16]: No need to redeclare these. They are exported below. We can avoid redundant code.
```
    <Default />
    <br />
    <Alt />
    <br />
    <Alt2 />
```

File: 01-atoms/buttons/buttons.stories.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Atoms/Button' };
+
+export const All = () => (
+  <>
+    <Button>Button Default</Button>
+    <br />
```
Comment (nJim) [inline 01-atoms/buttons/buttons.stories.js:13]: I don't have a design eye, but this 'All' story is very hard to look at with the buttons butted up against each other. Maybe not right now, but at some point, we could look for places to enhance our demo components.

**Comment by ModulesUnraveled** [comment]
Made all changes requested, consolidating to one PR here: https://github.com/emulsify-ds/western-up-react/pull/4

**Pull request closed without merging**